tests/services/test_pricing_service.py

```
@@ -200,3 +201,64 @@ def test_weekend_peak_with_extra(self, svc):
         # 기본 합계: 43000
         # 추가: 2명 * 5000 * 2h = 20000
         assert price == 63000
+
+
+class TestSeasonPricing:
+    """시즌 가격 적용 (A안: 크롤링 수집 season 태그는 활성으로 간주)"""
+
+    @patch.object(PricingService, '_is_season_active', return_value=True)
+    def test_season_rule_applied_on_match(self, mock_is_active, svc):
```

Ruff (0.15.2)

[warning] Unused method argument: `mock_is_active`

(ARG002)